### PR TITLE
add load from form to ES

### PR DESCRIPTION
**Pull request by emord**
@orangejenny

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 13 errors:

* corehq/pillows/mappings/app_mapping.py, line 131 - E501 line too long (131 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 132 - E501 line too long (158 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 133 - E501 line too long (185 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 134 - E501 line too long (193 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 135 - E501 line too long (187 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 136 - E501 line too long (187 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 137 - E501 line too long (184 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 138 - E501 line too long (159 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 139 - E501 line too long (164 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 140 - E501 line too long (158 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 141 - E501 line too long (156 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 142 - E501 line too long (158 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 143 - E501 line too long (130 > 115 characters)

**Comment by dimagimon** [comment]
There are 13 errors:

* corehq/pillows/mappings/app_mapping.py, line 132 - E501 line too long (131 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 133 - E501 line too long (158 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 134 - E501 line too long (185 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 135 - E501 line too long (193 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 136 - E501 line too long (187 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 137 - E501 line too long (187 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 138 - E501 line too long (184 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 139 - E501 line too long (159 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 140 - E501 line too long (164 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 141 - E501 line too long (158 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 142 - E501 line too long (156 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 143 - E501 line too long (158 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 144 - E501 line too long (130 > 115 characters)

File: corehq/pillows/mappings/app_mapping.py [hunk 1]
```diff
@@ -129,6 +129,19 @@
                                                                                                                                'preload': {'dynamic': False,
                                                                                                                                            'type': 'object'}},
                                                                                                                 'type': 'object'},
+                                                                                               'load_from_form': {'dynamic': False,
```
Comment (emord) [inline corehq/pillows/mappings/app_mapping.py]: copied it from `case_preload`

**Comment by orangejenny** [comment]
need to update https://github.com/dimagi/commcare-hq/blob/master/testapps/test_elasticsearch/tests/test_prod_es_indices.py#L38 ?

**Comment by dimagimon** [comment]
There are 13 errors:

* corehq/pillows/mappings/app_mapping.py, line 132 - E501 line too long (131 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 133 - E501 line too long (158 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 134 - E501 line too long (185 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 135 - E501 line too long (193 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 136 - E501 line too long (187 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 137 - E501 line too long (187 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 138 - E501 line too long (184 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 139 - E501 line too long (159 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 140 - E501 line too long (164 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 141 - E501 line too long (158 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 142 - E501 line too long (156 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 143 - E501 line too long (158 > 115 characters)
* corehq/pillows/mappings/app_mapping.py, line 144 - E501 line too long (130 > 115 characters)

**Comment by emord** [comment]
thanks. fixed it

**Comment by orangejenny** [comment]
looks fine, not going to wait on tests

**Pull request merged**